wgpu/gui/flexx.py

```diff
@@ -24,5 +24,8 @@ def _draw_frame_and_present(self):
         window.requestAnimationFrame(self._draw_frame_and_present)
         self.draw_frame()
 
+    def request_draw(self):
+        pass  # already drawing continuously
```

Comment:
shouldn't this be a call to window.requestAnimationFrame?

Reply:
Fixed